src/Servant/Common/BaseUrl.hs

```diff
@@ -32,7 +32,7 @@ import           Reflex.Dom.Core
 import           Text.Read
 
 
-type SupportsServantReflex t m = (Reflex t, TriggerEvent t m, PerformEvent t m, HasWebView (Performable m), MonadJSM (Performable m))
+type SupportsServantReflex t m = (Reflex t, TriggerEvent t m, PerformEvent t m, MonadJSM (Performable m))
```

Comment:
Looking at `reflex-dom-core`'s changelog, it looks like `HasWebView` was deleted just a couple weeks ago in the bump to 0.7.0. I think that means dropping them from `servant-reflex` forces us onto exactly `reflex-dom-core-0.7.0`.

Reply:
If you're curious about that change, here's the MR: https://github.com/reflex-frp/reflex-dom/pull/429/files

Reply:
got it!